### DISCUSSION TITLE
Setting up an URDF file, RVIZ not working

### DISCUSSION THREAD

**Discussion by sebastianfer** [iRobotEducation/create3_docs#111] [Questions]
I created an URDF file for my robot in my Virtual Machine. The launch command works and it opens RVIZ, but I can't visualize my robot. I get this error: _**No tf data. Actual error: Frame [base_link] does not exist.**_ 

Could someone explain me why this is happening and how I can solve it?

**Comment by shamlian** [comment]
Could you tell us a little more about your setup? What OS in your VM? What commands exactly are you running from opening a new terminal? Is there any setup going on in your .bashrc (if Linux)?
